src/main/java/it/gov/pagopa/payhub/auth/service/a2a/retreive/ClientDTOMapper.java

```diff
@@ -0,0 +1,17 @@
+package it.gov.pagopa.payhub.auth.service.a2a.retreive;
+
+import it.gov.pagopa.payhub.auth.model.Client;
+import it.gov.pagopa.payhub.model.generated.ClientDTO;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ClientDTOMapper {
+
+	public ClientDTO map(Client client) {
```

Comment:
A suggestione, in a micro-service, in order to keep it simple, it could be allowed to share the same class if the content are equal.
It's ok do it, but it would be also ok to just use the Client class itself

Reply:
I know, thanks for the advise anyway